Description: Disable unused-local-typedefs errors
 Disables new unused-local-typedefs as found in gcc 4.8
 .
 Based on commit to trunk upstream.
Author: James Page <james.page@ubuntu.com>
Origin: https://github.com/mongodb/mongo/commit/2d30f0b562c1925a52670e767c2c0222189fb8f0

--- a/src/third_party/v8/SConscript
+++ b/src/third_party/v8/SConscript
@@ -48,7 +48,8 @@
     'all': {
       'CCFLAGS':      ['-Wno-unused-parameter',
                        '-Woverloaded-virtual',
-                       '-Wnon-virtual-dtor']
+                       '-Wnon-virtual-dtor',
+                       '-Wno-unused-local-typedefs']
     },
     'mode:debug': {
       'CPPDEFINES':   ['ENABLE_DISASSEMBLER', 'DEBUG'],
